### PR TITLE
Deno 1.30 changes

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: getting_started/configuration_file.md [hunk 1]
```diff
@@ -6,22 +6,110 @@ compiler, formatter and linter.
...
+
+Then your script can use the bare specifier `std`:
+
+```js
+import { assertEquals } from "std/testing/assert.ts";
+
+assertEquals(1, 2);
+```
+
+The top-level `deno.json` option `importMap` along with the `--importmap` flag
```
Comment (aapoalas) [inline getting_started/configuration_file.md:35]: Should the precedence be documented? I think the workspace settings can also have an import map definition which, according to some LSP error messages I saw in the imports PR, might take precedence over all others? Or maybe that's only for the LSP?

File: getting_started/configuration_file.md [hunk 2]
```diff
@@ -6,22 +6,110 @@ compiler, formatter and linter.
...
 [Since v1.18](https://deno.com/blog/v1.18#auto-discovery-of-the-config-file),
 Deno will automatically detect `deno.json` or `deno.jsonc` configuration file if
-it's in your current working directory (or parent directories). To manually tell
-Deno to use a specific configuration file pass `--config path/to/file.json`
-flag.
-
-> ⚠️ Starting with Deno v1.22 you can disable automatic detection of the
-> configuration file, by passing `--no-config`.
-
-Note that using a configuration file is not required now, and will not be
```
Comment (aapoalas) [inline getting_started/configuration_file.md:16]: I guess this is now gone :D

Comment (ry) [reply getting_started/configuration_file.md:16]: It's not gone - it's just not so important that it needs to be here.

**Comment by ry** [comment]
closed in favor of #528

**Pull request closed without merging**